src/FastExpressionCompiler.LightExpression/Expression.cs

```diff
@@ -61,6 +61,9 @@ public abstract class Expression
 
         private static readonly ConstantExpression _nullExprInfo = new ConstantExpression(null, typeof(object));
 
+        public static NewExpression New(Type type) =>
+            new NewExpression(type.GetTypeInfo().DeclaredConstructors.First(x => x.GetParameters().Length == 0), Tools.Empty<Expression>());
```

Comment:
You may replace `First(..)` with `Tool.GetFirst(..)`, should be more performant.